ta/mk/build-user-ta.mk

```diff
@@ -0,0 +1,42 @@
+include mk/cleanvars.mk
+sm := $(lastword $(subst /, ,$(dir $(ta-mk-file))))
+sm-$(sm) := y
+
+# Select TA target (aka TA dev kit, when delivered)
+ta-target := $(strip $(if $(CFG_USER_TA_TARGET_$(sm)), \
+		$(filter $(CFG_USER_TA_TARGET_$(sm)), $(ta-targets)), \
+		$(default-user-ta-target)))
+
+ta-dev-kit-dir$(sm) := $(out-dir)/export-$(ta-target)
+# EARLY_TA_PATHS may hold paths to a TA in this directory
```

Comment:
Does this comment relate to the next line?

Reply:
It was only relevant in an earlier version of the patch set, I'll remove it.